_posts/2017-06-01-install-party.md

```diff
@@ -0,0 +1,13 @@
+---
+layout: post
+title: "Install Party + Social"
+slug: install-party
+text: 'tomesh monthly install party and social'
+location: Semaphore Studio 307, Claude T. Bissell, 140 St. George St  
+locationLink: http://osm.org/go/ZX6Bw~WNh--?m=
+date: 2017-06-01
```

Comment:
June is a long month. We periodically reset/added a week to keep us in the same week-of-month over the last year